XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Simulation/Hands/SimulatedHandControllerPose.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System;
+using System.Collections.Generic;
+using XRTK.Definitions.Utilities;
+using XRTK.Services;
+using XRTK.Definitions.Controllers.Simulation.Hands;
+using XRTK.Providers.Controllers.Hands;
+
+namespace XRTK.Providers.Controllers.Simulation.Hands
+{
+    /// <summary>
+    /// Simulatd pose of an hand controller defined by recorded joint poses.
+    /// </summary>
+    public class SimulatedHandControllerPose
```

Comment:
I think a lot of the implementation details in here should probably belong in the `SimulatedHandController` class